AutoMad/autoMad.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 22 13:14:59 2024
+
+@author: dragon
+"""
+
+import random
+import re
+import wordGrabber
```

Comment:
I like following a practice of separating out library imports differentiating third party libraries and files from your package

``` python
import random
import re
# ...

import wordGrabber
import madLibFeeder
# ...
```
It can also be nice to organize imports alphabetically within each group just to make it easier to keep track of things.

[isort](https://pycqa.github.io/isort/) will do this for you automatically.